cmake/CMakeLists.txt

```diff
@@ -1440,6 +1440,10 @@ if(onnxruntime_USE_OPENVINO)
 
   add_definitions(-DUSE_OPENVINO=1)
 
+  if (onnxruntime_DISABLE_GRAPH_PARTITION)
```

Comment:
the name onnxruntime_DISABLE_GRAPH_PARTITION makes it sound like an onnxruntime framework option.
alternatively,  could it be done as an openvino provider option at runtime, which is probably better for the user?

Reply:
please change to onnxruntime_OPENVINO_DISABLE_GRAPH_PARTITION